test/Microsoft.AspNetCore.Diagnostics.Tests/ExceptionHandlerTest.cs

```diff
@@ -176,6 +176,48 @@ public class ExceptionHandlerTest
         }
 
         [Fact]
+        public async Task Reexecute_RequestWithQueryString()
```

Comment:
Am I missing some inter-op between `UseStatusCodePagesWithReExecute` and `UseStatusCodePagesWithRedirects` or are you just saying that in general it's missing a test and I may as well add it in this PR?